Resources/views/DashboardAdmin/compose_container_show.html.twig

```diff
@@ -19,7 +19,7 @@
         <label>{{ 'composer.block.add.type'|trans({}, 'SonataDashboardBundle') }}</label>
         <select class="dashboard-composer__block-type-selector__select" style="width: auto">
             {% for blockServiceId, blockService in blockServices %}
-                <option value="{{ blockServiceId }}">{{ blockService.name }}</option>
+                <option value="{{ blockServiceId }}">{{ blockService.blockMetadata.title|trans({}, blockService.blockMetadata.domain|default('SonataBlockBundle')) }}</option>
```

Comment:
Yes, we are using the same mechanism on the page bundle: https://github.com/sonata-project/SonataPageBundle/blob/3.x/Resources/views/PageAdmin/compose_container_show.html.twig#L12